packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -56,17 +58,40 @@ def _estimate(operator: Estimatable, state: ITensorStateT) -> Estimate[complex]:
     op = convert_operator(operator, s)
 
     # calculate expectation value
-    psi = jl.apply(circuit, psi)
+    psi = jl.apply(circuit, psi, **kwargs)
     exp: float = jl.expectation(psi, op)
 
     return _Estimate(value=exp, error=0.0)
 
 
-def create_itensor_mps_estimator() -> QuantumEstimator[ITensorStateT]:
+def create_itensor_mps_estimator(
```

Comment:
It's better to  add `*` so that users can only pass [keyword-only arguments](https://peps.python.org/pep-3102/)?

Reply:
fixed in 
30f2833